src/collections/_documentation/workflow/notifications/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Notifications
+sidebar_order: 2
+---
+
+Notifications in Sentry can be routed to many supported [integrations]({%- link _documentation/workflow/integrations/index.md-%}), but by default are aimed at email. Notifications fall into two categories:
+
+[Alerts]({%- link _documentation/workflow/notifications/alerts.md -%})
+
+: Generated based upon a project’s alert rules. By default this is only when a new issue is seen.
+
+[Workflow]({%- link _documentation/workflow/notifications/workflow.md -%})
+
+: All activity related to user actions such as resolution, comments, and automatic regressions.
+
+Each category has its own set of configuration and subscription options. By default you’ll be subscribed to everything, but this can be changed via your [account settings](https://sentry.io/settings/account/notifications/). Read about [notification management]({%- link _documentation/workflow/notifications/notification-management.md -%}) for more information.
+
+## Issue States
+
+Sentry provides a few states for each event, which greatly impact how notifications work:
```

Comment:
"Sentry provides a few states for each event..."

Technically, the state belongs to the issue, not the event. So I'd suggest 

a) change "event" --> "issue" in the first two blurbs

b) make the third blurb something along the lines of:

"Regardless of alert rules, new events in ignored issues will not send alerts. (In addition, by default, ignored events are hidden in your project's issue stream. Search the issue stream for `is:ignored` to find them.)"